cmd/gitserver/server/cleanup.go

```diff
@@ -312,7 +339,14 @@ func (s *Server) freeUpSpace(howManyBytesToFree int64) error {
 
 	// Remove repos until howManyBytesToFree is met or exceeded.
 	var spaceFreed int64
+	mountPoint, err := findMountPoint(s.ReposDir)
+	if err != nil {
+		return errors.Wrap(err, "finding mount point")
+	}
 	for _, d := range gitDirs {
+		if spaceFreed >= howManyBytesToFree {
+			return nil
+		}
```

Comment:
```suggestion
		}
		
```
nit: https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/code_style_guide.md#use-new-lines-intentionally